smarts/core/controllers/trajectory_tracking_controller.py

```diff
@@ -208,14 +208,39 @@ def perform_trajectory_tracking_PD(
         lateral_gain = lerp(3, final_lateral_gain, normalized_speed)
         heading_gain = lerp(0.03, final_heading_gain, normalized_speed)
         steering_filter_constant = lerp(
-            2, final_steering_filter_constant, normalized_speed
+            12, final_steering_filter_constant, normalized_speed
         )
-        heading_error_derivative_gain = lerp(
+        heading_error_derivative_gain = 0 * lerp(
             1.5, final_heading_error_derivative_gain, normalized_speed
         )
```

Comment:
This seems to be invalidating the configuration. That is not necessarily wrong because the controller can interpret the configuration in any way it chooses but it is definitely controller configuration.

These changes to the controller appear to be overly specific.

Reply:
Yes, I agree.  That's how it was in the working (I assume) solution I saw in another repo, so that's why I brought that in here.  But I don't like it as is.  I think I'll use an if-block instead of multiplying by `0`, to make it clearer that the config values are not being used.

Reply:
ok, I tried to make the current situation a bit clearer (without changing the overall result).